sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -166,6 +167,15 @@ CosmosClientMetadataCachesSnapshot metadataCaches() {
         return this.state;
     }
 
+    CosmosClientBuilder regionScopedSessionCapturingEnabled(boolean isRegionScopedSessionCapturingEnabled) {
```

Comment:
I assume this is not public because we always want to enable by default eventually - and only allow this to be disabled durng initial testing?

Reply:
Yes - making this package-private allows for some baking in time.